File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        vector<int> freq(26, 0);
        int n = s.size();
        for(int i = 0; i < n; i++){
            freq[s[i] - 'a']++;
        }
        int min = 0;
        for(int i = 0; i < 26; i++){
            if(freq[i] == 0);
            else if(freq[i] % 2) min++;
            else min += 2;
        }
        return min;
    }
};

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.py
class Solution(object):
    def minimumLength(self, s):
        """
        :type s: str
        :rtype: int
        """
        arr = [0] * 26
        count = 0 
        for i in s:
            o = ord(i)
            arr[o - 97] += 1
        
        for i in arr:
            if i == 0:
                count += 0
            else:
                if i % 2 == 0:
                    count += 2
                else:
                    count += 1
        
        return count
        